articles/server-platforms/nodejs/_includes/_api_authz.md

```diff
@@ -0,0 +1,234 @@
+
+```jade
+// views/layout.jade
+
+doctype html
+html
+  head
+    title= title
+    link(rel='stylesheet', href='/stylesheets/style.css')
+    script(src="https://cdn.auth0.com/w2/auth0-7.2.js")
+  body
+    block content
+```
+
+Next, create the `index` view. It is here where you will instantiate the Auth0 client and create a function that will initiate the authentication process when the Login button is clicked by the user.
+
+```jade
+// views/index.jade
+
+extends layout
+
+block content
+
+  h1= Welcome to my application
+  br
+  button(onclick="signIn()") Login
+
+  script.
+    // Construct an instance of the auth0 client
+    var auth0 = new Auth0({
+      domain:       '#{env.AUTH0_DOMAIN}',
+      clientID:    '#{env.AUTH0_CLIENT_ID}',
+      callbackURL: '#{env.AUTH0_CALLBACK_URL}',
+    });
+    
+    // this function will be called when a user clicks the login button
+    function signIn() {
+      auth0.login({
+      responseType: 'id_token token',
+      scope: 'openid profile {API SCOPES}',
+      audience: '{API IDENTIFIER}'
+      });
+    }
+
+```
+
+The `audience` parameter should contain your API identifier from the Dashboard. If you don't send this, the runtime will take it from the tenant settings (`tenant.default_audience` or you can set it in the Dashboard). The `scope` parameter should include one or more scopes (separated by a space) you defined in the Dashboard for your API, in addition to any of the standard [openid scopes](https://auth0.com/docs/scopes).
+
+In `views/user.jade` we simply display the user's nickname and profile picture.
+
+```jade
+extends layout
+
+block content
+  img(src="#{user.picture}")
+  h2 Welcome #{user.nickname}!
+  br
+  a(href='/logout') Logout
+```
+
+## Making an Authenticated API Call
+
+Use the `access_token` to invoke the Resource Server (API):
+
+```js
+// invoke API endpoint with authn header
+var options = {
+  url: 'https://my.api/todo',
+  headers: {
+    'Authorization': 'Bearer ' + req.user.extraParams.access_token
+  }
+};
+
+request(options, function(error, response, body) {
+  var info = JSON.parse(body);
+  res.json({jwt: info.jwt});
+});
+
+```
+
+The Resource Server (API) should be configured to verify the JWT and any claims contained within it. Because the Resource Server is utilizing the RS256 signature method, tokens are signed using Auth0's private key for your account. Verification is done using the corresponding public key, which can be found at the following standard [JWKS (JSON Web Key set)](https://self-issued.info/docs/draft-ietf-jose-json-web-key.html) URL: https://${account.namespace}/.well-known/jwks.json. You can use any [recommended JWT library](https://jwt.io) to validate the standard claims returned in the token. These details are outside the scope of this quickstart tutorial. More information can be found [in our documentation](https://auth0.com/docs/api-auth/config/asking-for-access-tokens).
```

Comment:
maybe `More information can be found in the Quickstarts document for your API` ?